src/libsync/ownsql.cpp

```diff
@@ -303,6 +303,10 @@ void SqlQuery::bindValue(int pos, const QVariant& value)
                 res = sqlite3_bind_null(_stmt, pos);
             }
             break; }
+        case QVariant::ByteArray: {
+            auto ba = value.toByteArray();
+            res = sqlite3_bind_text(_stmt, pos, ba.constData(), ba.size(), SQLITE_TRANSIENT);
```

Comment:
(I meant because of encoding issues.. but since the data-fingerprint is from WebDAV it is probably anyway safe ascii or so)